pkg/piecestore/rpc/client/client.go

```diff
@@ -25,6 +25,11 @@ func New(ctx context.Context, conn *grpc.ClientConn) *Client {
 	return &Client{ctx, pb.NewPieceStoreRoutesClient(conn)}
 }
 
+// NewCustomRoute creates new Client with custom route interface
+func NewCustomRoute(ctx context.Context, route pb.PieceStoreRoutesClient) *Client {
+	return &Client{ctx, route}
```

Comment:
i missed this in the review for @aleitner, but it's definitely more robust from a future-changes perspective if you always name your struct values during creation and don't rely on positional order.

Reply:
Yes, you are right. Done.